docs/using-latest-daily.md

```diff
@@ -6,6 +6,21 @@ These instructions will get you set up with the latest build of Aspire. If you j
 
 See [machine-requirements.md](machine-requirements.md).
 
+## [Optional] Using scripts to install the latest .NET Aspire build from release branches
```

Comment:
is it obvious why you might want to do this? presumably on a particular day, the release branch is very close to the last public release. so it would be something like - you want to validate a port of a change? might be worth a note making clear that almost certainly you want main

Reply:
also given this is the 1% case maybe it should go at the end of the doc